### ISSUE TITLE
Suggestion: save embeds, etc into your discord account

### ISSUE THREAD

**Issue by CnyAnime** [discohook/site#23]
Dear Discohook Developer,

My suggestion is to save all json (or Embed data) into the Discord account.

With kind regards,
Crunchy

**Comment by wolfier74** [comment]
Good idea or create username and password login system where you can save your data

**Comment by vvvvvvivi** [comment]
> Good idea or create username and password login system where you can save your data

it will use login with Discord instead

**Comment by wolfier74** [comment]
Ok

**Comment by CnyAnime** [comment]
> Good idea or create username and password login system where you can save your data

the create username and password login system isn't a good idea because just use with Discord

**Comment by BadCoder1337** [comment]
I think that a simple "cloud" synchronization is a lightweight and flexible solution. It's trivial to build it on top of the current backup system. You could just login with Discord via OAuth (e.g. using very flexible `next-Auth`) and upload current local backups to the server. After login, on a fresh browser app just fetch latest backups from the server. 

Backups could be synchronized one-by-one or entirely synchronized per user.The first case requires additional DB entities, but we can build private sharing system.In the second case we can store all backups in the simple key-value storage (Redis).

**Comment by vivisrc** [comment]
I got a separate backend in the works not under source control right now with a full user account system and much more.

**Comment by CnyAnime** [comment]
> I got a separate backend in the works not under source control right now with a full user account system and much more.

Will you make a Beta Version of Discohook?

**Comment by vivisrc** [comment]
> Will you make a Beta Version of Discohook?

I don't understand what you mean precisely but like the redesign changes I'll keep it away from the main domain for a while so people can test it out before I push it to everyone.

The site will likely need a full rewrite given how many design and logic changes there are, so even after the API exists I'll need some time before it's usable by the public.

**Comment by CnyAnime** [comment]
> > Will you make a Beta Version of Discohook?
> 
> I don't understand what you mean precisely but like the redesign changes I'll keep it away from the main domain for a while so people can test it out before I push it to everyone.
> 
> The site will likely need a full rewrite given how many design and logic changes there are, so even after the API exists I'll need some time before it's usable by the public.

Will there a Beta phase of the newest version of Discohook. That mean's before you publish the public version the users can test it and search some Bugs and then they report that to GitHub and you will fix it.

**Comment by vivisrc** [comment]
> Will there a Beta phase of the newest version of Discohook. That mean's before you publish the public version the users can test it and search some Bugs and then they report that to GitHub and you will fix it. 

Yes, just like the redesign of the site last year

**Comment by CnyAnime** [comment]
> > Will there a Beta phase of the newest version of Discohook. That mean's before you publish the public version the users can test it and search some Bugs and then they report that to GitHub and you will fix it.
> 
> Yes, just like the redesign of the site last year

I see
